modules/client-configuration/pages/clients-slmicro.adoc

```diff
@@ -49,9 +69,25 @@ include::snippets/addchannels_vendor_webui.adoc[]
 Alternatively, you can add channels at the command prompt.
 The channels you need for this procedure are:
 
-[[slmicro-channels-cli]]
+// 6.1 from command line
+[[slmicro-channels-cli-61]]
+[cols="1,1", options="header"]
+.{sle-micro} {microversion} Products - CLI
```

Comment:
```suggestion
.{sl-micro} {microversion} Products - CLI
```
Nor sure but it probably should be sl here?

Reply:
ah thanks, typo.